pages/install/conda-install.rst

```diff
@@ -45,32 +47,87 @@ obtained are specified by adding the ``--channel`` option in the ``install`` or
 
-To use the scripts and modules installed in the ``spam`` environment,
-you must activate it it by running::
+To use the scripts and modules installed in the ``ct-env`` environment, including Jupyter,
+you must activate it it by running:
+
+.. code:: shell
+
+   conda activate ct-env
+
+**Option 2: Create a new environment using an environment file**
+
+This option is similar to **Option 1** but includes a few other packages that
+you may find helpful as you're working with Cantera. Copy and paste the contents
+of the file shown below into a file called ``environment.yaml``. Then, save the
+the file somewhere and remember that location.
+
+.. code:: yaml
+
+   name: ct-env
+   channels:
+   - cantera
+   - defaults
+   dependencies:
+   - python
+   - cantera
+   - ipython  # optional (needed for nicer interactive command line)
+   - jupyter  # optional (needed for Jupyter Notebook)
+   - matplotlib  # optional (needed for plots)
+   - python-graphviz  # optional (needed for reaction path diagrams)
+   - h5py  # optional (needed for HDF/H5 output)
+   - pandas  # optional (needed for pandas interface)
+
+From the command line (or the
+Anaconda Prompt on Windows), change directory into the folder where you saved
+``environment.yaml``:
+
+.. code:: shell
+
+   cd folder/where/you/saved
 
-    conda activate spam
+and then run:
 
-**Option 2: Install Cantera in an existing environment**
+.. code:: shell
 
-First, activate your environment (assumed here to be named ``baked_beans``; if you've
+   conda env create -f environment.yaml
+
+This will create an environment called ``ct-env``. Once you've done that, you
+need to activate the environment before using any scripts or modules that you
+just installed:
+
+.. code:: shell
+
+   conda activate ct-env
+
+**Option 3: Install Cantera in an existing environment**
+
+First, activate your environment (assumed here to be named ``ct-env``; if you've
 forgotten the name of the conda environment you wanted to use, the command
-``conda env list`` can help). This is done by running::
+``conda env list`` can help). This is done by running:
 
-    conda activate baked_beans
+.. code:: shell
 
-Then, install Cantera in the active enironment by running::
+   conda activate ct-env
 
-    conda install --channel cantera cantera
+Then, install Cantera in the active enironment by running:
 
-**Option 3: Install the development version of Cantera**
+.. code:: shell
 
-To install a recent development snapshot (i.e., an alpha or beta version) of
-Cantera in an existing environment, activate the environment and then run::
+   conda install --channel cantera cantera
 
-    conda install --channel cantera/label/dev cantera
+**Option 4: Install the development version of Cantera**
+
+To install a recent development snapshot (that is, an alpha or beta version) of
+Cantera in an existing environment, activate the environment and then run:
+
+.. code:: shell
+
+   conda install --channel cantera/label/dev cantera
```

Comment:
`ct-dev` to differentiate again? If the comments in `environment.yaml` could be left in, it would dove-tail nicely.